docs/config-openshift-v4_19-exp.md

```diff
@@ -1,12 +1,12 @@
 ---
 # This file is automatically generated from internal/doc and Ignition's
 # config/doc.  Do not edit.
-title: OpenShift v4.18.0-experimental
```

Comment:
OK, I said to split the docs change in a fourth commit but that makes things hard to review so let's push the doc changes in each commit instead.